test/jdk/java/awt/Robot/HiDPIScreenCapture/ScreenCaptureGtkTest.java

```diff
@@ -40,27 +46,26 @@
  *           Gtk backends and presence of UI scaling
  * @requires os.family == "linux"
  * @run main/othervm -Djdk.gtk.version=2 -Dsun.java2d.uiScale=1 ScreenCaptureGtkTest
- * @run main/othervm -Djdk.gtk.version=2 -Dsun.java2d.uiScale=2 ScreenCaptureGtkTest
- * @run main/othervm -Djdk.gtk.version=2 -Dsun.java2d.uiScale=3 ScreenCaptureGtkTest
  * @run main/othervm -Djdk.gtk.version=3 -Dsun.java2d.uiScale=1 ScreenCaptureGtkTest
- * @run main/othervm -Djdk.gtk.version=3 -Dsun.java2d.uiScale=2 ScreenCaptureGtkTest
- * @run main/othervm -Djdk.gtk.version=3 -Dsun.java2d.uiScale=3 ScreenCaptureGtkTest
  */
 
 public class ScreenCaptureGtkTest {
     private static final Color[] COLORS = {
             Color.GREEN, Color.BLUE, Color.ORANGE, Color.RED};
 
     public static void main(String[] args) throws Exception {
+        final int topOffset = 50;
+        final int leftOffset = 50;
+
         Frame frame = new Frame();
-        // Position the frame on prime number coordinates to avoid
-        // them being multiple of the desktop scale; this tests Linux
-        // color picker better.
+        // Position the frame such that color picker will work with
+        // prime number coordinates to avoid them being multiple
+        // of the desktop scale; this tests Linux color picker better.
         // Also, the position should be far enough from the top left
         // corner of the screen to reduce the chance of being repositioned
         // by the system because that area's occupied by the global
         // menu bar and such.
-        frame.setBounds(83, 97, 400, 300);
+        frame.setBounds(89, 99, 100, 100);
```

Comment:
99 is not a prime number. Is it intentional?

Reply:
Now that there's an offset within the frame, the numbers should take that into account: (50+89)==139 and (50+99)==149 and these are prime numbers.